src/styles/components/pods-view.less

```diff
@@ -0,0 +1,76 @@
+.pod-history-table {
+
+  tbody {
+    line-height: 1.8;
```

Comment:
Let's use the canvas variables for line height definitions:
https://github.com/mesosphere/canvas/blob/master/styles/variables/variables-tables.less#L2269

NB: The reference is to a newer canvas then what we are currently using in dcos-ui!